news/2020-12-09-new-featured-artist-lifetheory.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Lifetheory"
+date: 2020-12-09 09:00:00 +0000
+---
+
+**Lifetheory** joins our Featured Artist lineup as our latest addition, and that's a fact!
+
+![](https://assets.ppy.sh/artists/122/header.jpg)
+
+Fusing a whole slew of genres into something new and wonderful, **Lifetheory** is an insanely talented Russian producer sporting a similarly wide range of musical influences. Drawing elements from the likes of *Pendulum*, *Hanz Zimmer* and *Linkin Park*, his music effortlessly strides into aural territory rarely tread.
+
+With **19** new tracks now available, there's no shortage of epic electronic experiences waiting in the wings, so why not give some a try?
+
+In case you weren't with us yet when the **Lifetheory** craze was at its peak, check it all out in a little medley style preview below, courtesy of the most pishi of fats:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/122/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Still need more convincing? We've got you covered - check out the trilogy of maps that started it all:
+
+### Daisy
+
+Check out [this classic 2013 set](https://osu.ppy.sh/beatmapsets/118494) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/122/Daisy/Lifetheory%20-%20Daisy.mp3" type="audio/mpeg">
+</audio>
+
+### Angel
+
+Check out another entry in the trilogy with [this classic 2013 set](https://osu.ppy.sh/beatmapsets/113192) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/122/Daisy/Lifetheory%20-%20Angel.mp3" type="audio/mpeg">
+</audio>
+
+### Fairy
+
+Check out the final entry in the trilogy with [this classic 2014 set](https://osu.ppy.sh/beatmapsets/113192) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/122/Daisy/Lifetheory%20-%20Fairy.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+Like what you see/hear? Check out [**Lifetheory**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/122) for a whole load more, all pre-packaged and ready for some life via the application of mapping theory.
+
+That's all for today, but we'll be back again (as usual) on Saturday with another awesome artist, so make sure to stay in the know! The [@osugame Twitter](https://twitter.com/osugame) is an excellent place to do that, if you're not following it already.
+
+—Ephemeral & pishifat
```

Comment:
```suggestion
—Ephemeral and pishifat
```
i know it looks a little worse here, but the more prominent top of news page will end up reading "by Ephemeral and pishifat". + consistency with other posts